### DISCUSSION TITLE
Unable to send resource content for review by a specific user

### DISCUSSION THREAD

**Discussion by sravani-64** [Sunbird-Ed/Community#537] [Bugs]
Hello Community,
We are facing an issue in production environment running on Sunbird 4.4 version. **A specific user with **content creator role** is able to create course, collection and send it for review.
But the user is not able to send created resource or uploaded content for review.**
Below errors are shown in network tab as
![image](https://github.com/Sunbird-Ed/Community/assets/133327173/7dca5006-4aed-493a-87b9-c39b2ea68b0f)

**Other users with same role are able to send all content for review.** Could You please help us how to resolve this issue.
Thank you

**Comment by tsprasath** [comment]
@vinukumar-vs Can you help on it.

cc: @gandham-santhosh

**Reply by Krishnaj20** [reply]
@rajeevsathish - can you check this query and respond?

**Reply by rajeevsathish** [reply]
@sravani-64 this does not look like the issue as the same issue is there in the staging env of sunbird 
will see what is the issue with this api and try to fix but content not able to send for review is not because of this issue 
please can you take some time and see in the logs what is the error you are getting
for your reference
`curl 'https://staging.sunbirded.org/content-editor/undefined?6.0.0.08e5381' \
  -H 'Accept: */*' \
  -H 'Accept-Language: en-GB,en;q=0.9' \
  -H 'Connection: keep-alive' \
  -H 'Cookie: connect.sid=s%3A6zvQ-ZLRzWYopE_KfavmWOZvf1lf6ryS.x0OWAn9SvUC6SXQmW19Lhu7gIdhaEn7WNjJafQLk6RU' \
  -H 'Referer: https://staging.sunbirded.org/content-editor/index.html?6.0.0.08e5381' \
  -H 'Sec-Fetch-Dest: script' \
  -H 'Sec-Fetch-Mode: no-cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36' \
  -H 'sec-ch-ua: "Not.A/Brand";v="8", "Chromium";v="114", "Google Chrome";v="114"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "macOS"' \
  --compressed
`

**Reply by rajeevsathish** [reply]
@sravani-64 hope this issue is fixed?

**Reply by chitranshu-keshav** [reply]
@sravani-64 Please get back in case the issue is not resolved

**Comment by sravani-64** [comment]
@rajeevsathish @chitranshu-keshav apologies for the delay, but the developers team is still looking into it and we are waiting for their update

**Reply by chitranshu-keshav** [reply]
@sravani-64 any update on this issue? If the issue is resolved then can you mark this thread as answered
cc @ashokreddy1208
